plugins/woocommerce/templates/myaccount/form-login.php

```diff
@@ -12,7 +12,7 @@
  *
  * @see     https://woocommerce.com/document/template-structure/
  * @package WooCommerce\Templates
- * @version 9.2.0
+ * @version 9.6.0
```

Comment:
```suggestion
 * @version 9.7.0
```

I'm bumping this to `9.7.0`, as the code freeze for `9.6.0` was a few days ago.